openlcs/packages/views.py

```diff
@@ -889,6 +889,10 @@ class CheckDuplicateImport(APIView):
     def post(self, request, *args, **kwargs):
         results = dict()
         data = request.data
+        parent = data.get('parent', '')
+        if parent != '':
+            return Response(data={
+                    'results': results})
```

Comment:
This way, the package deduplication can not apply to the same component from different containers/rpmmods, which makes the duplicate checking much less useful.